robotpy/resources/py/preexec.py

```diff
@@ -16,9 +16,9 @@
 is_windows = hasattr(sys, 'getwindowsversion')
 
 # setup minimum requirements here 
-PYFRC_MIN_VERSION = LooseVersion("2015.2.0")
-WPILIB_MIN_VERSION = LooseVersion("2015.0.6")
-NETWORKTABLES_MIN_VERSION = LooseVersion("2015.0.6")
+PYFRC_MIN_VERSION = LooseVersion("2017.1.5")
+WPILIB_MIN_VERSION = LooseVersion("2017.3.1")
```

Comment:
There is no 3.1 is there?

Reply:
There is a 3.1 It came out in may afaik

Reply:
Is this....min robotpy version or wpilibj version?